build.gradle

```diff
@@ -1,5 +1,39 @@
+plugins {
+	id 'com.moowork.node' version '0.13'
+}
+
+node {
+	version = '6.9.1'
+	npmVersion = '3.10.8'
+    workDir file("rcp/org.testeditor.web.testexecutionlogview/node")
```

Comment:
Ok fixed it. 
I used visual studio code for this file and the web project. It uses for tabs spaces. I changed my config for this.